docker-compose.dev.yml

```diff
@@ -4,7 +4,7 @@ services:
       - ${WIS2BOX_HOST_DATADIR}:/data/wis2box:rw
       - ./wis2box-management/wis2box/wis2box.cron:/etc/cron.d/wis2box:ro
       - ./wis2box-management/wis2box:/app/wis2box
-    command: ["wis2box", "pubsub" , "subscribe", "--broker", "http://wis2box-minio:9000", "--topic", "wis2box-storage/#", "--verbosity", "INFO"]
+    command: ["wis2box", "pubsub" , "subscribe"]
```

Comment:
Options no longer required?

Reply:
the options were mostly not used inside subscribe.py . The topic used for subscription can not be a configurable at this time as it set in other places (such as acl.conf for mosquitto, notification-topic for minio) and subscribing to any other topic than 'wis2box/#' would breake the wis2box-workflow.